components/platform_crypto/shave_trusted_boot/firmware.c

```diff
@@ -26,6 +261,13 @@ uint64_t c_MEASURE_SIZE() /*$ cn_function MEASURE_SIZE; $*/ { return MEASURE_SIZ
 #if defined(WITH_ATTEST) || defined(CN_ENV)
 // must go in special protected storage (writable only by firmware/hardware)
 static byte last_measure[MEASURE_SIZE];  // initial contents unimportant
+#if defined(NO_XMSS)
+static bool public_key;
+static bool xmss_signature;
+#else
+static XmssPublicKey public_key;
+static XmssSignatureBlob xmss_signature;
```

Comment:
Having a hardcoded key makes sense, but having a hardcoded signature doesn't (if you're going to hardcode the XMSS signature of a particular input, you could just as well hardcode its SHA256 hash and save yourself a lot of trouble).  I think it would make more sense for `xmss_signature` to be a local inside `main`/`reset`, initialized from part of the input (e.g. last N bytes are the signature and everything before that is the code being signed).

Reply:
I put the signature in the expected measure